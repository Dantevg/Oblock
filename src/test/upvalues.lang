{
	a = "global"
	{
		showA = () => {
			print a
		}
		showA()
		a = "block"
		showA()
	}
}
