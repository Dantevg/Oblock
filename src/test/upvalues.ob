{
	a = "global"
	var b
	{
		show = () => print(a, b)
		show()
		print(a, b)
		var a = "block"
		var b = "B"
		show()
		print(a, b)
	}
	print(a, b)
}
