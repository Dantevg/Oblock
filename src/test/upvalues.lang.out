global	nil
global	nil
global	nil
block	B
global	nil
